.github/workflows/on-pullrequest-dockerfile.yml

```diff
@@ -18,6 +18,8 @@ env:
   DOCKER_HUB_TOKEN: ${{ secrets.DOCKER_HUB_TOKEN }}
   IMAGE_NAME: dms
 
+permissions: read-all
+
```

Comment:
@msilesgap there is one failure now that needs to be addressed, please see https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/actions/runs/8740650184/job/23984809261?pr=39

Reply:
I've set the write perrmission to the pull-request